### PR TITLE
feat: make Conan quiet if cmake is not verbose

### LINKED ISSUE #12: If CMake is quiet, Conan should be quiet too
The verbosity levels should be passed to Conan as is in CMake.

### PR DESCRIPTION
**Pull request by aminya**
Fixes #12

### CODE REVIEW

File: src/Conan.cmake [hunk 1]
```diff
@@ -32,6 +32,11 @@ macro(run_conan)
     set(LIST_OF_BUILD_TYPES ${CMAKE_CONFIGURATION_TYPES})
   endif()
 
+  is_verbose(_is_verbose)
+  if(NOT ${_is_verbose})
+    set(CONAN_QUIET YES)
+  endif()
```
Comment (ddalcino) [inline src/Conan.cmake:38]: I am unclear about the consequences of this when `_is_verbose` is true. As far as I can tell, the CONAN_QUIET variable will just be an empty string when you try to use it. Should it not be set to “NO” or “OFF” or something?

Comment (lefticus) [reply src/Conan.cmake:38]: Agreed, it does seem like we need something different here

Comment (aminya) [reply src/Conan.cmake:40]: Fixed in https://github.com/cpp-best-practices/project_options/pull/79/commits/fa2266813748dcdbd4dd335d112b0344333eca27

**Comment by aminya** [comment]
@lefticus @ddalcino please test this locally on your system. For me, Conan is so fragile that I cannot make it work.

**Comment by aminya** [comment]
The tests show that this is working as expected.

**Pull request merged**